packages/react/src/utils/cardUtilityFunctions.js

```diff
@@ -92,9 +92,7 @@ export const compareGrains = (grain1, grain2) => {
 
 export const getUpdatedCardSize = (oldSize) => {
```

Comment:
If its needed for value card may be you can do it specifically for value card. looks like getUpdatedCardSize function getting used in almost all the cards
![image](https://github.com/user-attachments/assets/8c927026-eedd-4d9f-9af4-e742907bdf53)